### DISCUSSION TITLE
Error: Element type is invalid - next-9-4v

### DISCUSSION THREAD

**Discussion by danilockthar** [vercel/next.js#14050] [Help] [answered]
Hello, i just created a new project without knowing there is a new version. I created my folders and my pages , my _app and my _document but im not getting the new error overlay, it doesnt show me ( like the previous of 9.3 ) where is my error. 

Its getting harder to make a simple next app now. 

Im receiving: 

```
Server Error
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

This error happened while generating the page. Any console logs will be displayed in the terminal window.
```

```
Call Stack
ReactDOMServerRenderer.render
file:///C:/xampp/htdocs/daniphp/nextjs/greatone/node_modules/react-dom/cjs/react-dom-server.node.development.js (3743:17)
ReactDOMServerRenderer.read
file:///C:/xampp/htdocs/daniphp/nextjs/greatone/node_modules/react-dom/cjs/react-dom-server.node.development.js (3373:29)
renderToString
file:///C:/xampp/htdocs/daniphp/nextjs/greatone/node_modules/react-dom/cjs/react-dom-server.node.development.js (3988:27)
render
file:///C:/xampp/htdocs/daniphp/nextjs/greatone/node_modules/next/dist/next-server/server/render.js (3:298)
```

**Steps to reproduce it**

1- Create new dir , then install next react react-dom . 

2- Populate your project dir with pages folder, create an _app.js and _document.js along with an index.js and a Layout component

3- Run: npm run dev

**Comment by rafaelalmeidatk** [comment, marked as answer]
Hi, can you share a repo with a minimal reproduction? These steps aren't enough/clear enough to reproduce the error. Looks like you are incorrectly importing some component, thus the `undefined` component error message.

**Reply by danilockthar** [reply]
Here is the repo :) . Its a simple new project with one page and three component, its weird that i cant find the missimported or exported component, i start new projects like this and never see this error.

https://github.com/danilockthar/portfolio

Steps to reproduce it:

Download the repository, and command `npm install `   then `npm run dev ` in your command line.

**Reply by rafaelalmeidatk** [reply]
@danilockthar the first thing I did was resetting the `_app.js` to the defaults, and the app worked, so the error was there. Then I reverted back to your code and removed `ZEITUIProvider`, the code worked. This component isn't exported from `@zeit-ui/react`, from the [docs](https://github.com/zeit-ui/react#quick-start), the correct component name is `ZeitProvider`

**Reply by danilockthar** [reply]
> @danilockthar the first thing I did was resetting the `_app.js` to the defaults, and the app worked, so the error was there. Then I reverted back to your code and removed `ZEITUIProvider`, the code worked. This component isn't exported from `@zeit-ui/react`, from the [docs](https://github.com/zeit-ui/react#quick-start), the correct component name is `ZeitProvider`

Oh they changed the component name ? because im having that component working on another projects and they worked fine. 
Any chance to the new error overlay to show that type of error in the future ? I mean, to comunicate the error were especifically in _app.js or something like that ? right now the stack trace points to ReactDomServer . 

Anyway, its working, thank you very much!
